### ISSUE TITLE
Add bin files

### ISSUE THREAD

**Issue by sabuto** [Patrick-Thomas/Sparkpad-Arduino#2]
I would suggest adding the bin files in the format of releases for people that dont really know (or want to) compile from scratch. Personally i would suggest doing it with Github Actions

**Comment by Patrick-Thomas** [comment]
Do you mean so people can work outside the Arduino ecosystem? As this repo is in the form of an Arduino library I don't think it makes sense for that sort of stuff to be here, maybe a separate repo would make more sense?

By utilising the Arduino IDE we don't have to worry about building and compiling ourselves and can make use of all the libraries Arduino has to offer, so unless there's some specific functionality we're missing I'd prefer to stick to this format.

**Comment by sabuto** [comment]
No, I mean if you upload the bin files which is what get's put onto the board it will enable people to upload to the board without nessaserily having to have the arduino or platformio on their system if they dont want to work on the code itself. take a look at https://github.com/Aircoookie/WLED/releases/tag/v0.11.1 for example

**Comment by Patrick-Thomas** [comment]
Gotcha. You would still need to download avrdude and use the command line to get that bin onto the board though. I'd always prefer to just do it through the IDE, but I am an Arduino fanboy.

I'll add the bin to the releases then we're covering all the bases
